PackageInfo.g

```diff
@@ -50,6 +50,10 @@ Date := "18/07/2018",
 ##  <!ENTITY RELEASEDATE "18 July 2018">
 ##  <#/GAPDoc>
 
+## Optional: license of the package, as an SPDX short-form identifiers;
+## see <https://spdx.org/ids>
```

Comment:
Looks good, but maybe point to https://spdx.org/licenses/ ?

Reply:
Well, I pointed at /ids because this explains what an SPDX ID is, resp. an "SPDX short-form identifiers", and then links to the license list. While on the license list, this is not immediately visible.

Reply:
I've now expanded the comment substantially